release/rancher/rancher.go

```diff
@@ -234,9 +234,14 @@ func createDashboardReferencesPR(ctx context.Context, cfg *config.Dashboard, ghC
 	}
 
 	// creating a pr from your fork branch
-	_, _, err := ghClient.PullRequests.Create(ctx, cfg.RancherRepoOwner, cfg.RancherRepoName, pull)
+	pr, _, err := ghClient.PullRequests.Create(ctx, cfg.RancherRepoOwner, cfg.RancherRepoName, pull)
+	if err != nil {
+		return err
+	}
 
-	return err
+	fmt.Println("Pull Request created successfully:", pr.GetHTMLURL())
```

Comment:
Add a space at the end after the colon.

Reply:
`fmt.Println(...)` already adds one space between each param: https://go.dev/play/p/MeEk6aGetU_d